### PR TITLE
Chart and StockChart documentation

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/stockchart/labels-template-and-format.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+position: 22
+---
+
+# Label Template and Format
+
+The StockChart for Blazor can render labels on the axes, and the navigator. You can control those texts not only through the values you data bind, but also through [format strings](#format-strings) or [templates](#templates).
+
+You can also rotate the labels by setting the `Angle` parameter to a numeric value the labels will rotate to.
+
+* [Format Strings](#format-strings)
```
Comment (yordan-mitev) [inline components/stockchart/labels-template-and-format.md:17]:
```suggestion
In this article:
* [Format Strings](#format-strings)
```

File: components/stockchart/labels-template-and-format.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        public decimal Low { get; set; }
+
+        public int Volume { get; set; }
+    }
+}
+````
+
+In a **category axis label template**, you can use the following fields:
+
+* `value` - the category value
```
Comment (yordan-mitev) [inline components/stockchart/labels-template-and-format.md:229]:
```suggestion
* `value` —the category value
```

File: components/stockchart/labels-template-and-format.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+* `format` - the default format of the label
```
Comment (yordan-mitev) [inline components/stockchart/labels-template-and-format.md:230]:
```suggestion
* `format`—the default format of the label
```

File: components/stockchart/labels-template-and-format.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+
+<!--* `dataItem` - the data item, in case a field has been specified. If the category does not have a corresponding item in the data then an empty object will be passed.-->
+<!--* culture - the default culture (if set) on the label-->
+
+In a **value axis label template**, you can use the following fields:
+
+* `value` - the label value
```
Comment (yordan-mitev) [inline components/stockchart/labels-template-and-format.md:237]:
```suggestion
* `value`—the label value
```

File: components/stockchart/labels-template-and-format.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+        public decimal High { get; set; }
+
+        public decimal Low { get; set; }
+
+        public int Volume { get; set; }
+    }
+}
+````
+
+In a **category axis label template**, you can use the following fields:
```
Comment (yordan-mitev) [inline components/stockchart/labels-template-and-format.md:227]: Don't use bold for names or emphasis. This seems to be a proper name, so you can apply the corresponding capitalization: Category Axis Label Template

File: components/stockchart/labels-template-and-format.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+
+* `value` - the category value
+* `format` - the default format of the label
+
+<!--* `dataItem` - the data item, in case a field has been specified. If the category does not have a corresponding item in the data then an empty object will be passed.-->
+<!--* culture - the default culture (if set) on the label-->
+
+In a **value axis label template**, you can use the following fields:
```
Comment (yordan-mitev) [inline components/stockchart/labels-template-and-format.md:235]: Don't use bold.

**Review by dimodi — APPROVED** [review]

**Pull request merged**